src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -19,7 +19,7 @@ namespace ov {
 namespace genai {
 
 // Return flag corresponds whether generation should be stopped: false means continue generation, true means stop.
-using StreamerVariant = std::variant<std::function<bool(std::string)>, std::shared_ptr<StreamerBase>, std::monostate>;
+using StreamerVariant = std::variant<std::function<bool(std::string)>, std::function<StreamerRunningStatus(std::string)>, std::shared_ptr<StreamerBase>, std::monostate>;
```

Comment:
Add a deprecation notice.
You can also print a warning from `create_streamer()`.

Reply:
I have added comment here and OPENVINO_DEPRECATED to create_streamer function. Warning seems more logical to me, but it will trigger on each generate and it looks too much. What do you think is better way ?